src/client/components/app/Transactions/WithdrawTx.tsx

```diff
@@ -212,7 +223,7 @@ export const WithdrawTx: FC<WithdrawTxProps> = ({ header, onClose, children, ...
 
   const txActions = [
     {
-      label: t('components.transaction.approve'),
+      label: signedApprovalsEnabled ? t('components.transaction.sign') : t('components.transaction.approve'),
```

Comment:
Should we have just 1 button that says `sign & withdraw`? I think it provides a better ux for the user

Reply:
This UX was discussed before in Design phase and ended as its currently handled